config/globals.js

```diff
@@ -83,22 +70,30 @@ const globals = {
 	TEST_HASH: (process.env.TEST_HASH || '').trim(),
 
 	// files
-	FILE_PREVIEW_SERVICE_URI: process.env.FILE_PREVIEW_SERVICE_URI || 'http://localhost:3000/filepreview',
-	FILE_PREVIEW_CALLBACK_URI: process.env.FILE_PREVIEW_CALLBACK_URI
-		|| 'http://localhost:3030/fileStorage/thumbnail/',
-	ENABLE_THUMBNAIL_GENERATION: process.env.ENABLE_THUMBNAIL_GENERATION || false,
-	FILE_SECURITY_CHECK_SERVICE_URI: process.env.FILE_SECURITY_CHECK_SERVICE_URI
-		|| 'http://localhost:8081/scan/file',
+	FILE_PREVIEW_SERVICE_URI:
+		process.env.FILE_PREVIEW_SERVICE_URI ||
+		'http://localhost:3000/filepreview',
+	FILE_PREVIEW_CALLBACK_URI:
+		process.env.FILE_PREVIEW_CALLBACK_URI ||
+		'http://localhost:3030/fileStorage/thumbnail/',
+	ENABLE_THUMBNAIL_GENERATION:
+		process.env.ENABLE_THUMBNAIL_GENERATION || false,
+	FILE_SECURITY_CHECK_SERVICE_URI:
+		process.env.FILE_SECURITY_CHECK_SERVICE_URI ||
```

Comment:
Codacy found an issue: ['||' should be placed at the beginning of the line.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)